api/HTMLMediaElement.json

```diff
@@ -305,6 +411,15 @@
               "notes": [
                 "Firefox doesn't implement this yet. See <a href='https://bugzil.la/847377'>bug 847377</a>."
               ]
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
```

Comment:
Like you've done above, could you add the `null` Safari (desktop and iOS) entries?

Reply:
Added